convene-web/spec/tasks/release_after_build_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe "rake release:after_build" do
+  subject(:run_rake_task!) do
+    ConveneWeb::Application.load_tasks
+    Rake::Task["release:after_build"].reenable
```

Comment:
We probably don't need this line but if we intended to have multiple it block, this is needed.

Reply:
Seems useful regardless!